src/styles/_Grid.scss

```
@@ -5,13 +5,19 @@
 
 .container {
   margin: 0 auto;
-  max-width: $grid-max-width;
+  max-width: $container-max-width;
   padding: 0 2rem;
   position: relative;
   width: 100%;
 
   &.small {
-    max-width: $container-max-width-sm;
+    max-width: $content-max-width;
+  }
+}
+
+@media (min-width: $breakpoint-medium) {
+  .container {
+    width: (10/12) * 100%;
```

Can utilize sass' percentage here which compiles to the same. See https://www.sassmeister.com/gist/857243520f42d4526fa4e7588353cb61?token=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&scope=gist,read:user

```suggestion
    width: percentage(10/12);
```